### PR TITLE
Vcon 52 banner

### PR DESCRIPTION
**Pull request by shewood**
[vcon-52]

Added the banner to the header.

### CODE REVIEW

File: src/components/atoms/Banner.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          uppercase
+          bg-blue-deep bg-opacity-95
+          text-white
+          inline-block
+          whitespace-normal
+        "
+      >
+        {{ title }}
+      </h1>
+      <div class="">
```
Comment (nibivi77) [inline src/components/atoms/Banner.vue:20]: Can we remove this div and add CSS to the p tag instead?

File: src/components/molecules/TheHeader.vue [hunk 2]
```diff
@@ -120,16 +121,21 @@
           text="menu"
         ></Menu>
       </div>
+      <div class="sm:py-4">
```
Comment (nibivi77) [inline src/components/molecules/TheHeader.vue:124]: Can we discard this div and apply the sm:py on the wrapper div inside the "TheHeader" component?

**Review by nibivi77 — COMMENTED** [review]
This looks good following the Figma. The mobile does not look the same but I assume the banner is subject to change.

Let me know what you think about the comments and I will approve.

File: src/components/atoms/Banner.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="bg-banner-img bg-cover p-10">
+    <div class="mx-auto container px-3">
+      <h1
```
Comment (DaceyTom2) [inline src/components/atoms/Banner.vue:4]: The h1 and p tag overlap slightly leaving a dark line
image

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: src/components/atoms/Banner.vue (hunk 1 above)
Comment (shewood) [reply src/components/atoms/Banner.vue:20]: i took out the empty class but can't get rid of the div.

File: src/components/atoms/Banner.vue (hunk 3 above)
Comment (shewood) [reply src/components/atoms/Banner.vue:4]: yes, it is overlapping.

**Review by DaceyTom2 — APPROVED** [review]
Looks beautiful!

**Pull request merged**